API.md

```diff
@@ -571,13 +571,13 @@ Returns an instance of [Identity](https://github.com/orbitdb/orbit-db-identity-p
 ```javascript
 const identity = db.identity
 console.log(identity.toJSON())
-{ id: 'QmZyYjpG6SMJJx2rbye8HXNMufGRtpn9yFkdd27uuq6xrR',
-  publicKey: '0446829cbd926ad8e858acdf1988b8d586214a1ca9fa8c7932af1d59f7334d41aa2ec2342ea402e4f3c0195308a4815bea326750de0a63470e711c534932b3131c',
-  signatures:
+{ _id: 'QmZyYjpG6SMJJx2rbye8HXNMufGRtpn9yFkdd27uuq6xrR',
+  _publicKey: '0446829cbd926ad8e858acdf1988b8d586214a1ca9fa8c7932af1d59f7334d41aa2ec2342ea402e4f3c0195308a4815bea326750de0a63470e711c534932b3131c',
+  _signatures:
    { id: '3045022058bbb2aa415623085124b32b254b8668d95370261ade8718765a8086644fc8ae022100c736b45c6b2ef60c921848027f51020a70ee50afa20bc9853877e994e6121c15',
      publicKey: '3046022100d138ccc0fbd48bd41e74e40ddf05c1fa6ff903a83b2577ef7d6387a33992ea4b022100ca39e8d8aef43ac0c6ec05c1b95b41fce07630b5dc61587a32d90dc8e4cf9766'
    },
-  type: 'orbitdb'
+  _type: 'orbitdb'
```

Comment:
reading from the source they shouldnt have the _underscores 
https://github.com/orbitdb/orbit-db-identity-provider/blob/e93809dda6cfa5b0fd843815d2a5f8f5bc1e81d9/src/identity.js#L79-L86